src/transformGDocs/styleGDocs.ts

```diff
@@ -0,0 +1,33 @@
+export function transformGDocsStyles(html: string): string {
+    html = html.replace(/(&nbsp;|\u00A0)/g, ' ');
+    const parser = new DOMParser();
+    const doc = parser.parseFromString(html, 'text/html');
+    const blackColors = /^#000000$|^#000$|^rgb\(\s*0\s*,\s*0\s*,\s*0\)$|^black$/i;
+    doc.querySelectorAll('[style]').forEach(node => {
+        const el = node as HTMLElement;
+        const style = el.getAttribute('style') || '';
```

Comment:
It should be possible to access the styles via the style API here: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style